packages/grid/x-data-grid-pro/src/tests/cellEditing.DataGridPro.test.tsx

```diff
@@ -591,6 +591,34 @@ describe('<DataGridPro /> - Cell Editing', () => {
     expect(cell).to.have.class('MuiDataGrid-cell--editing');
   });
 
+  it('should be able to type in a custom input', () => {
```

Comment:
To be more precise about what the test is doing.

```suggestion
  it('should not call .preventDefault on key events inside a cell in edit mode', () => {
```